app/team_test.go

```diff
@@ -460,3 +460,92 @@ func TestAddUserToTeamByHashMismatchedInviteId(t *testing.T) {
 	assert.Nil(t, team)
 	assert.Equal(t, "api.user.create_user.signup_link_mismatched_invite_id.app_error", err.Id)
 }
+
+func TestJoinUserToTeam(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	id := model.NewId()
+	team := &model.Team{
+		DisplayName: "dn_" + id,
+		Name:        "name" + id,
+		Email:       "success+" + id + "@simulator.amazonses.com",
+		Type:        model.TEAM_OPEN,
+	}
+
+	if _, err := th.App.CreateTeam(team); err != nil {
+		t.Log(err)
+		t.Fatal("Should create a new team")
+	}
+
+	maxUsersPerTeam := th.App.Config().TeamSettings.MaxUsersPerTeam
+	defer func() {
+		th.App.UpdateConfig(func(cfg *model.Config) { cfg.TeamSettings.MaxUsersPerTeam = maxUsersPerTeam })
+		th.App.SetDefaultRolesBasedOnConfig()
```

Comment:
This line was accidentally included (I don't know how the tests passes in the branch where it was merged).

Reply:
@jespino anything I can do to help with this one?

Reply:
Not really, we only need to remove that line, I can enter in the branch and make the commit.

Reply:
Done!, I have removed the line, I'm not sure if this will fix the merge, but at least, will fix one of the problems.